enterprise-suite/alertmanager/slack.tmpl

```diff
@@ -1,4 +1,3 @@
 {{ define "title.slack" }}{{ .GroupLabels.namespace}}/{{ .GroupLabels.es_workload }} [{{ .Status | toUpper }}][{{ .CommonLabels.severity | toUpper }}]{{ end }}
-{{ define "text.slack" }}{{ range $index, $element := .Alerts }}_{{ .Annotations.summary }}_
-    {{ .Annotations.description }}
+{{ define "text.slack" }}{{ range $index, $element := .Alerts }}_{{ .CommonLabels.name }}_
```

Comment:
I assume .CommonLabels.name is the monitor name?  If so, lgtm

Reply:
Seems like it's not? At least, @srikanthops is having issues with it.

Reply:
Oh I see the issue - .CommonLabels isn't available in the range. It should be `.Labels.name`.